install.md

```diff
@@ -1,17 +1,23 @@
-Installing Kubecost
+Installation
 ===================
 
-We **recommend using the [Kubecost helm chart](http://kubecost.com/install) to install and operate Kubecost**. This install method is available for free and contains all the required components to get started, provides access to all Kubecost features, and can scale to large clusters. It also provides the most flexibility for configuring Kubecost and its dependencies.
+To getting started with OpenCost or Kubecost, **the recommended path is to [install Kubecost community version](http://kubecost.com/install)**. This install method is available for free and leverages the Kubecost helm chart. It provides access to all Community Kubecost functionality and can scale to large clusters. This will also provide a token for trialing and retaining data across different Kubecost product tiers.
```

Comment:
In addition to @teevans comment, I'd maybe go with `OpenCost and Kubecost` instead of `or`, because one builds on top of the other

Reply:
great points! 

> Maybe 'To get started' would flow a little better

updated. 
> Also thoughts on using 'access to all OpenCost functionality'?

What about "OpenCost and Community Kubecost functionality?"

> In addition to @teevans comment, I'd maybe go with OpenCost and Kubecost instead of or, because one builds on top of the other

updated. i guess technically you could get started with OpenCost but not Kubecost, so and/or would be most correct?